components/chainHoldings.js

```diff
@@ -1,43 +1,24 @@
-import { useEffect, useState } from 'react';
-import formatToDollar from '../utils/helpers'
-import {chainMapping} from '../utils/tokenMapping'
+import { useEffect, useState } from 'react'
+import formatToDollar from '@/utils/helpers'
+import { chainMapping } from '@/utils/tokenMapping'
+import ListCard from '@/components/library/ListCard'
 
 const ChainHoldings = () => {
-  const [data, setData] = useState(null);
+  const [data, setData] = useState(null)
 
   useEffect(() => {
     fetch('/api/data?file=sums.csv&type=summary')
-      .then(response => response.json())
-      .then(data => setData(data))
-      .catch(error => console.error('Error:', error));
-  }, []);
+      .then((response) => response.json())
+      .then((data) => {
+        const sortedData = Object.entries(data)
```

Comment:
Doing the sorting and whatnot only once, on load, when data is pulled is more efficient than doing it on render.